stories/basic.stories.tsx

```diff
@@ -17,6 +36,7 @@ const BasicSchema = z.object({
   address: z.object({
     city: Cities,
   }),
+  testArray: ArraySchema.array().min(2),
```

Comment:
```suggestion
  children: Child.array(),
```

Reply:
I believe the .min() is important to show that you can not only check for array items rules but also you can add a global array rule